src/network-config.js

```diff
@@ -23,6 +23,7 @@ export const networkConfigs = {
       'https://api.thegraph.com/subgraphs/name/aragon/aragon-mainnet',
     settings: {
       chainId: 1,
+      networkType: 'main',
       ...KNOWN_CHAINS.get(1),
```

Comment:
Can we make this property, `testnet = true | false`? the reason is we already have networkType = 'main' in the wallet context which can be confusing if we have another networkType = 'main' which means a different thing.
Also, I see this value as being true or false, not any other value and we don't need to do string comparison in network.js..